### ISSUE TITLE
Regression: WebView2 keyboard navigation with Tab key not getting registered  (accessibility issue)

### ISSUE THREAD

**Issue by lukajustic** [MicrosoftEdge/WebView2Feedback#3585]
**Description**
On WebView2 (version 114.0.1823.51) the tab key is not registered and a customer cannot navigate the app using tab key. This is bad for accessibility.

**Version**
SDK: Microsoft.UI.Xaml 2.8.4
Runtime: 114.0.1823.51
Framework: Win32
OS: Win10

**Regression**
Yes, last time I tested this in 114.0.1823.37 there was no issue

**Repro Steps**
I used demo **MyUWPGetStartApp** from here: https://github.com/MicrosoftDocs/edge-developer/blob/main/microsoft-edge/webview2/get-started/winui2.md

1.) Open the demo app
2.) Click on "Tab" key. 
3.) It should start putting elements in focus yet it doesn't

Interesting observation:  
- Once I right click and open dev tools the tabbing starts working. However closing and opening the app will once again make tabbing not work
- Also minimising the app and then un-minimising also makes the tab start working

[AB#45395518](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/45395518)

**Comment by vagisha-nidhi195** [comment]
To add to the weird behaviour, it also starts working when you minimize and re-open the app. Seem like this has regressed recently and is a big issue for any webview2 based app affecting accessibility.

**Comment by novac42** [comment]
Thanks for reporting the issue. I've assigned this to a dev that can help follow up on this.

**Comment by tofuandeve** [comment]
Hi  I tried this on our UWP test app with Webview2 Runtime version 114.0.1823.58 and the issue doesn't seem to repro. Could you try it out on the later version of Webview Runtime to see if this issue still repro for you?

**Comment by lukajustic** [comment]
We've tested this issue on 114.0.1823.58 on multiple devices using both demo MyUWPGetStartApp and our custom app. The issue is still there.

Make sure you do not minimise the app after first launch as minimisation makes the issue go away. The problem is with first app load.

**Comment by lukajustic** [comment]
Yes this is exactly the experience I would expect, and no, there were no bounding boxes. We tried on Microsoft.UI.Xaml 2.8.4 with 114.0.1823.58 runtime on Windows 10 device.

**Comment by tofuandeve** [comment]
@lukajustic Hey, I got the issue to repro on the UWP MyUWPGetStartApp  sample app. This issue seems to be related to another issue that we have with WinUI2. This is issue is being investigated in the current sprint for us.

**Comment by tofuandeve** [comment]
hi @lukajustic We're talking to WinUI team to figure out the root cause of this issue right now. Could you do a quick check with webview2 runtime version 114.0.1823.37, I see that you mention the issue wasn't there for that version, but it seems to repro for me for that runtime version and the versions before that.

**Comment by maitreyee06** [comment]
hi @tofuandeve, me and my team are working on a project, and seem to be running into this same issue. Do you know when this issue will be fixed?

**Comment by VishyBala007** [comment]
Any update on this same issue I am also facing
first time tabs dont work, then open anything else then come back it works.

Once lost focus fires then got focus fire then only it works..
By default you launch it wont work..

**Comment by champnic** [comment]
This is fixed in WinAppSDK/WinUI3 version 1.6+. The WinUI team isn't doing updates to UWP WebView2 anymore, so unfortunately I don't think we'll get a fix for this in UWP.
